code/client/clrcore/External/VehicleHash.cs

```diff
@@ -287,78 +477,134 @@ public enum VehicleHash : uint
 		Nimbus = 2999939664u,
 		Ninef = 1032823388u,
 		Ninef2 = 2833484545u,
+		Niobe = 1881415402u,
+		Nokota = 1036591958u,
+		Novak = 2465530446u,
 		Omnis = 3517794615u,
+		OmniseGT = 3789743831u,
+		Openwheel1 = 1492612435u,
+		Openwheel2 = 1181339704u,
 		Oppressor = 884483972u,
+		Oppressor2 = 2069146067u,
 		Oracle = 1348744438u,
 		Oracle2 = 3783366066u,
 		Osiris = 1987142870u,
+		Outlaw = 408825843u,
 		Packer = 569305213u,
+		Panthere = 2100457220u,
 		Panto = 3863274624u,
 		Paradise = 1488164764u,
+		Paragon = 3847255899u,
+		Paragon2 = 1416466158u,
+		Paragon3 = 3348919626u,
+		Pariah = 867799010u,
 		Patriot = 3486509883u,
+		Patriot2 = 3874056184u,
+		Patriot3 = 3624880708u,
+		PatrolBoat = 4018222598u,
 		PBus = 2287941233u,
+		PBus2 = 345756458u,
 		PCJ = 3385765638u,
 		Penetrator = 2536829930u,
 		Penumbra = 3917501776u,
+		Penumbra2 = 3663644634u,
 		Peyote = 1830407356u,
+		Peyote2 = 2490551588u,
+		Peyote3 = 1107404867u,
 		Pfister811 = 2465164804u,
 		Phantom = 2157618379u,
 		Phantom2 = 2645431192u,
 		Phantom3 = 177270108u,
+		Phantom4 = 4165683409,
 		Phoenix = 2199527893u,
 		Picador = 1507916787u,
 		Pigalle = 1078682497u,
+		Pipistrello = 4071505793u,
+		Pizzaboy = 1968807591u,
+		PolDominator10 = 3521165271u,
+		PolDorado = 2666966727u,
+		PolGauntlet = 3061199846u,
+		PolGreenwood = 1737348074u,
 		Police = 2046537925u,
 		Police2 = 2667966721u,
 		Police3 = 1912215274u,
 		Police4 = 2321795001u,
+		Police5 = 2620582743u,
 		Policeb = 4260343491u,
 		PoliceOld1 = 2758042359u,
 		PoliceOld2 = 2515846680u,
 		PoliceT = 456714581u,
+		PoliceT3 = 2850111293u,
+		PolImpaler5 = 1249425552u,
+		PolImpaler6 = 1452003510u,
 		Polmav = 353883353u,
 		Pony = 4175309224u,
 		Pony2 = 943752001u,
+		Postlude = 4000288633u,
 		Pounder = 2112052861u,
+		Pounder2 = 1653666139u,
+		Powersurge = 2908631255u,
 		Prairie = 2844316578u,
 		Pranger = 741586030u,
 		Predator = 3806844075u,
 		Premier = 2411098011u,
+		Previon = 1416471345u,
 		Primo = 3144368207u,
 		Primo2 = 2254540506u,
 		PropTrailer = 356391690u,
 		Prototipo = 2123327359u,
+		Pyro = 2908775872u,
+		R300 = 1076201208u,
 		Radi = 2643899483u,
+		Raiden = 2765724541u,
+		Raiju = 239897677u,
 		RakeTrailer = 390902130u,
 		RancherXL = 1645267888u,
 		RancherXL2 = 1933662059u,
 		RallyTruck = 2191146052u,
 		RapidGT = 2360515092u,
 		RapidGT2 = 1737773231u,
+		RapidGT3 = 2049897956u,
 		Raptor = 3620039993u,
 		RatBike = 1873600305u,
+		Ratel = 3758861739u,
 		RatLoader = 3627815886u,
 		RatLoader2 = 3705788919u,
+		RCBandito = 4008920556u,
 		Reaper = 234062309u,
 		Rebel = 3087195462u,
 		Rebel2 = 2249373259u,
+		Rebla = 83136452u,
+		Reever = 1993851908u,
 		Regina = 4280472072u,
+		Remus = 1377217886u,
 		RentalBus = 3196165219u,
+		Retinue = 1841130506u,
+		Retinue2 = 2031587082u,
+		Revolter = 3884762073u,
 		Rhapsody = 841808271u,
+		Rhinehart = 2439462158u,
 		Rhino = 782665360u,
+		Riata = 2762269779u,
 		Riot = 3089277354u,
+		Riot = 22601952180u,
```

Comment:
```suggestion
		Riot2 = 2601952180u,
```

Small mistake here :)